src/System.Net.Mail/ref/System.Net.Mail.cs

```diff
@@ -81,12 +81,12 @@ public enum DeliveryNotificationOptions
     }
     public partial class LinkedResource : System.Net.Mail.AttachmentBase
     {
-        public LinkedResource(System.IO.Stream contentStream) : base (default(string)) { }
-        public LinkedResource(System.IO.Stream contentStream, System.Net.Mime.ContentType contentType) : base (default(string)) { }
-        public LinkedResource(System.IO.Stream contentStream, string mediaType) : base (default(string)) { }
-        public LinkedResource(string fileName) : base (default(string)) { }
-        public LinkedResource(string fileName, System.Net.Mime.ContentType contentType) : base (default(string)) { }
-        public LinkedResource(string fileName, string mediaType) : base (default(string)) { }
+        public LinkedResource(System.IO.Stream contentStream) : base (default(System.IO.Stream)) { }
+        public LinkedResource(System.IO.Stream contentStream, System.Net.Mime.ContentType contentType) : base (default(System.IO.Stream)) { }
+        public LinkedResource(System.IO.Stream contentStream, string mediaType) : base (default(System.IO.Stream)) { }
+        public LinkedResource(string fileName) : base (default(System.IO.Stream)) { }
+        public LinkedResource(string fileName, System.Net.Mime.ContentType contentType) : base (default(System.IO.Stream)) { }
+        public LinkedResource(string fileName, string mediaType) : base (default(System.IO.Stream)) { }
```

Comment:
Why is the type changing from 'string' to 'System.IO.Stream' here?  This now doesn't match the actual implementation.